aws/resource_aws_emr_cluster.go

```diff
@@ -532,6 +576,10 @@ func resourceAwsEMRClusterRead(d *schema.ResourceData, meta interface{}) error {
 		log.Printf("[ERR] Error setting EMR Ec2 Attributes: %s", err)
 	}
 
+	if err := d.Set("kerberos_attributes", flattenEmrKerberosAttributes(d, cluster.KerberosAttributes)); err != nil {
+		log.Printf("[ERR] Error setting EMR Kerberos Attributes: %s", err)
```

Comment:
I know the others aren't doing it, but it feels weird to me that this doesn't result in an error returning out of the read?

Reply:
Updated!